src/tests/ftest/rebuild/no_cap.py

```diff
@@ -154,8 +154,8 @@ def test_rebuild_no_capacity(self):
         # Check for pool and rebuild info after rebuild
         self.log.info("..(8)Verify pool and rebuild info after rebuild")
         pool_checks["pi_ndisabled"] = ">0"
-        rebuild_checks["rs_obj_nr"] = ">0"
-        rebuild_checks["rs_rec_nr"] = ">0"
+        rebuild_checks["rs_obj_nr"] = "=>0"
+        rebuild_checks["rs_rec_nr"] = "=>0"
```

Comment:
These comparison keys for the `_check_info()` method are incorrect:
```suggestion
        rebuild_checks["rs_obj_nr"] = ">=0"
        rebuild_checks["rs_rec_nr"] = ">=0"
```